### PR TITLE
Add readme info on the currently supported Apps

### PR DESCRIPTION
**Pull request by Waterdrips**
Signed-off-by: Alistair Hey <alistair@heyal.co.uk>

## Description

Add a list of apps we can install to the readme 
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,21 @@ k3sup app install --help
 k3sup app install APP_NAME --help
 ```
 
+Our currently supported apps:
+* OpenFaaS
```
Comment (alexellis) [inline README.md]: Should this have the names as per the install command?

File: README.md [hunk 2]
```diff
@@ -154,6 +154,21 @@ k3sup app install --help
...
+* nginx-ingress
+* cert-manager
+* openfaas-ingress
+* inlets-operator
```
Comment (alexellis) [inline README.md]: Is some of this duplicated in another section of the file?

Comment (Waterdrips) [reply README.md]: The section just above this list (part of this "section") has some further examples of a couple of the apps, this is just a bullet list of all the commands.

**Comment by alexellis** [comment]
I thought it may be listed around here -> https://github.com/alexellis/k3sup#-install-an-app-with-k3sup

**Comment by Waterdrips** [comment]
> I thought it may be listed around here -> https://github.com/alexellis/k3sup#-install-an-app-with-k3sup

https://github.com/Waterdrips/k3sup/tree/readme-add-apps#-install-an-app-with-k3sup

This pr adds a list of the apps supported at the bottom of that section, just below the section that shows how to get info on the k3sup apps

Screenshot 2019-12-13 at 11 25 16

**Pull request merged**